sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokers.java

```diff
@@ -180,8 +180,8 @@ private static Implementation delegateOrNoop(DoFnSignature.DoFnMethod method) {
   }
 
   /**
-   * Implements an invoker method ("instrumented method") by delegating to a method of the target
-   * {@link DoFn} ("target method").
+   * Implements a method of {@link DoFnInvoker} (the "instrumented method") by delegating to a
+   * "target method" of the wrapped {@link DoFn}.
```

Comment:
Could we also put something like this in Javadoc (on the `targetMethod` field, and the `instrumentedMethod` parameters)? That makes it easier to see in the UI and such.